content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,67 @@
 Invoice Time Spent on Tickets (Postpaid Support Services)
 =========================================================
 
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect your Helpdesk ticket to a Project task and use either one to record timesheets that can be
+later pulled from to bill your client. This integration provides more control and transparency over
+what the client is charged for.
 
 Configuration
 =============
 
-| Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk team --> Edit` and enable the options
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+Click **Edit** and enable the options *Timesheet on Ticket* and *Time Reinvoicing*.
 
-.. image:: media/reinvoice_time1.png
+In the *Timesheet on Ticket* drop-down option, choose the Project to which tickets (and timesheets)
+will be linked by default. Open the Project's external link to edit its settings and enable the
+*Billable* option. Save both the Project and the Helpdesk team's settings to apply these features.
```

Comment:
Same here. 
Should `*Billable*` be `:guilabel:` since it's a clickable option on the UI?

Reply:
Also changed to `:guilabel:` here.